content/doc/upgrade-guide/2.46.adoc

```diff
@@ -0,0 +1,19 @@
+---
+layout: documentation
+title:  Jenkins Upgrade Guide
+notitle: true
+---
+
+== Upgrading to Jenkins LTS 2.46.x
+
+Each section covers the upgrade from the previous LTS release, the section on 2.46.1 covers the upgrade from 2.32.3.
+
+=== Upgrading to Jenkins LTS 2.46.1
+
+==== JNLP4 remoting protocol enabled by default
+
+The JNLP4 protocol is now enabled by default for JNLP agent connections. It's more reliable than JNLP3, and also encrypted, making it the best choice for agent connections. Use of this protocol may require upgrades of +slave.jar+ files on agents.
+
+==== Obsolete ciphers removed from SSHD
+
+Jenkins exposes an SSH server that allows use of CLI commands via SSH. In this SSH server, support for the obsolete ciphers AES128CBC, TripleDESCBC, and BlowfishCBC has been removed.
```

Comment:
Add: "SSH clients using these versions won't be able to connect to Jenkins. It should not affect modern systems"